src/index.css

```diff
@@ -47,6 +47,12 @@ body {
       0 0/2rem 2rem;
   }
 }
+@media (hover: hover) and (pointer: fine) {
```

Comment:
This will check, if the browser supports real hover (mobile only emulate it) and will apply `:hover` for desktops

Reply:
And no, it's impossible to do `lg:hover:text-tertialry`, `active:text-tertiary`, we can use `useBreakpoints` for that, but I think this is more native way, because some desktops do support touchscreens (which will emulate `:hover` too)